frontend/test/playwright/utils/test.ts

```diff
@@ -0,0 +1,61 @@
+// eslint-disable-next-line no-restricted-imports
+import { test as base } from "@playwright/test"
+
+const encoder = new TextEncoder()
+
+const signingSecret = process.env.HMAC_SIGNING_SECRET
```

Comment:
We need to add this env variable to the infrastructure repository to get this to work, right?

Reply:
It's the same secret used for signing requests in k6, so no need for a new secret (though you could want to change the name to make it generic instead of k6 specific). See here in the new workflow file that I passed `K6_SIGNING_SECRET` for `HMAC_SIGNING_SECRET`:

https://github.com/WordPress/openverse/pull/5075/files#diff-c0447faa897e95e3c385302da0a0b3ea051e6c90be93e10cf0914eb8589a881dR34